src/README.md

```diff
@@ -8,13 +8,28 @@ The Timestream datasource plugin provides a support for [Amazon Timestream](http
 1. Click the **Add data source** button.
 1. Select **Timestream** in the **Time series databases** section.
 
-
 ## Authentication
 
-The Timestream plugin authentication system matches the standard Cloudwatch plugin system.  See [the grafana documentation](https://grafana.com/docs/grafana/latest/datasources/cloudwatch/#authentication) for authentication options and setup.
+For authentication options and configuration details, see [AWS authentication](https://grafana.com/docs/grafana/latest/datasources/aws-cloudwatch/aws-authentication/) topic.
+
+### IAM policies
 
+Grafana needs permissions granted via IAM to be able to read data from the Timestream API. You can attach these permissions to the IAM role or IAM user configured in the previous step.
 
-Once authentication is configured, click "Save and Test" to verify the service is working. Once this is configured, you can specify default values for the configuration.
+Here is a minimal policy example:
```

Comment:
I don't think we should call this "minimal" since grants permissions for everything (probably too much for grafana). I would say that instead and point to the AWS docs for more examples: https://docs.aws.amazon.com/timestream/latest/developerguide/security_iam_id-based-policy-examples.html#security_iam_id-based-policy-examples-common-operations

Reply:
Yeah good idea! Will update the docs